### DISCUSSION TITLE
Is it possible to clone a device with HAA firmware?

### DISCUSSION THREAD

**Discussion by juansf99** [RavenSystem/esp-homekit-devices#1811] [General]
Let's say I have a Shelly 1, with the latest HAA firmware, its JSON loaded and added in Homekit. If I take this Shelly 1, hook it up with FTDI, and make a copy of the firmware in a .bin file; then I take an original Shelly 1 and load this .bin into it... will the new cloned device work?

**Comment by xrust83** [comment]
Theoretically, to test this theory will take you 10-15 minutes. I had a similar experience with sonoff dual. When you copy one device and fill it in another, it comes to life and works in the Ewelink app 
https://youtu.be/Wl4NWKEBwFM
Since the mass address is cloned, the router even gives the same IP address, but in your case I understand you want to connect devices at the same time and this may be a failure, as the same SP will not get
